zc_install/includes/template/templates/index_default.php

```diff
@@ -110,7 +110,10 @@
 <?php } ?>
 <?php if (!$hasFatalErrors && !$hasWarnErrors && $hasUpdatedConfigFile) { ?>
     <div class="alert-box success">
-    <?php  echo TEXT_ERROR_SUCCESS_NO_ERRORS; ?>
+    <?php echo TEXT_ERROR_SUCCESS_NO_ERRORS; ?>
+    </div>
+    <div class="alert-box">
+        <?php echo TEXT_UPGRADE_INSTALL_INFO; ?>
```

Comment:
Does this properly disappear if no db already exists? 
ie: if no Upgrade option is available, telling them what the non-visible Upgrade button "does" isn't useful, and the text explaining "Clean Install" is probably alarming.

Reply:
If there are no configures, it all disappears as the error report is based on a configure.
![Clipboard01](https://user-images.githubusercontent.com/4391026/79343118-4c16b080-7f2e-11ea-83fa-862090826a05.gif)
Maybe better to split the information to a title/popup on each button?